docs/learn-crosschain.md

```diff
@@ -60,3 +90,8 @@ will include this block for parachain B into the Relay Chain.
 
 - [XCMP Scheme](https://research.web3.foundation/en/latest/polkadot/XCMP.html) - Full technical
   description of cross-chain communication on the Web3 Foundation research wiki.
+- [Messaging Overview](https://w3f.github.io/parachain-implementers-guide/messaging.html) - An
+  overview of the messaging schemes from the Parachain Implementor's guide.
+- [XCMP Format](https://github.com/paritytech/xcm-format) - Description of the XCMP format.
+
+[xcmp-format]: https://github.com/paritytech/xcm-format
```

Comment:
```suggestion
[xcm-format]: https://github.com/paritytech/xcm-format
```
Not sure if you meant XCMP format or XCM format based on the link attached

Reply:
Good catch!